File: src/main/resources/statics/js/modules/small/smallorder.js
$(function () {
    $("#jqGrid").jqGrid({
        url: baseURL + 'small/smallorder/list',
        datatype: "json",
        colModel: [			
			{ label: 'id', name: 'id', index: 'id', width: 50, key: true },
			{ label: '下单渠道', name: 'channel', index: 'channel', width: 80 }, 			
			{ label: '订单号', name: 'orderNo', index: 'order_no', width: 80 }, 			
			{ label: '用户id', name: 'userId', index: 'user_id', width: 80 }, 			
			{ label: '店铺id', name: 'supplierId', index: 'supplier_id', width: 80 }, 			
			{ label: '0生成、处理中、已完成', name: 'orderStatus', index: 'order_status', width: 80 }, 			
			{ label: '0未支付、1支付中、2已支付', name: 'payStatus', index: 'pay_status', width: 80 }, 			
			{ label: '0生成、1、退款中、2已退款', name: 'refundStatus', index: 'refund_status', width: 80 }, 			
			{ label: '0未发货、1发货中、2已发货', name: 'shipStatus', index: 'ship_status', width: 80 }, 			
			{ label: '商品(sku)原始价总额', name: 'skuOriginalTotalPrice', index: 'sku_original_total_price', width: 80 }, 			
			{ label: '商品(sku)现价总额', name: 'skuTotalPrice', index: 'sku_total_price', width: 80 }, 			
			{ label: '运费', name: 'freightPrice', index: 'freight_price', width: 80 }, 			
			{ label: '代金券优惠价', name: 'couponPrice', index: 'coupon_price', width: 80 }, 			
			{ label: '代金券id', name: 'couponId', index: 'coupon_id', width: 80 }, 			
			{ label: '实付订单金额', name: 'actualPrice', index: 'actual_price', width: 80 }, 			
			{ label: '支付金额', name: 'payPrice', index: 'pay_price', width: 80 }, 			
			{ label: '支付流水id(本地支付流水)', name: 'payId', index: 'pay_id', width: 80 }, 			
			{ label: '支付渠道名称', name: 'payChannel', index: 'pay_channel', width: 80 }, 			
			{ label: '支付时间(支付成功回调获取的支付时间)', name: 'payTime', index: 'pay_time', width: 80 }, 			
			{ label: '物流方式(物流方式代号)', name: 'shipCode', index: 'ship_code', width: 80 }, 			
			{ label: '物流单号', name: 'shipNo', index: 'ship_no', width: 80 }, 			
			{ label: '本地支付单号（用于与第三方支付交互）', name: 'paySn', index: 'pay_sn', width: 80 }, 			
			{ label: '发货时间', name: 'shipTime', index: 'ship_time', width: 80 }, 			
			{ label: '确认收货时间', name: 'confirmTime', index: 'confirm_time', width: 80 }, 			
			{ label: '省', name: 'province', index: 'province', width: 80 }, 			
			{ label: '市', name: 'city', index: 'city', width: 80 }, 			
			{ label: '县', name: 'county', index: 'county', width: 80 }, 			
			{ label: '详细地址', name: 'address', index: 'address', width: 80 }, 			
			{ label: '', name: 'memo', index: 'memo', width: 80 }, 			
			{ label: '', name: 'refundReason', index: 'refund_reason', width: 80 }, 			
			{ label: '', name: 'consignee', index: 'consignee', width: 80 }, 			
			{ label: '', name: 'phone', index: 'phone', width: 80 }, 			
			{ label: '', name: 'createTime', index: 'create_time', width: 80 }, 			
			{ label: '', name: 'modifyTime', index: 'modify_time', width: 80 }			
        ],
		viewrecords: true,
        height: 385,
        rowNum: 10,
		rowList : [10,30,50],
        rownumbers: true, 
        rownumWidth: 25, 
        autowidth:true,
        multiselect: true,
        pager: "#jqGridPager",
        jsonReader : {
            root: "page.list",
            page: "page.currPage",
            total: "page.totalPage",
            records: "page.totalCount"
        },
        prmNames : {
            page:"page", 
            rows:"limit", 
            order: "order"
        },
        gridComplete:function(){
        	//隐藏grid底部滚动条
        	$("#jqGrid").closest(".ui-jqgrid-bdiv").css({ "overflow-x" : "hidden" }); 
        }
    });
});

var vm = new Vue({
	el:'#icloudapp',
	data:{
		showList: true,
		title: null,
		smallOrder: {}
	},
	methods: {
		query: function () {
			vm.reload();
		},
		add: function(){
			vm.showList = false;
			vm.title = "新增";
			vm.smallOrder = {};
		},
		update: function (event) {
			var id = getSelectedRow();
			if(id == null){
				return ;
			}
			vm.showList = false;
            vm.title = "修改";
            
            vm.getInfo(id)
		},
		saveOrUpdate: function (event) {
		    $('#btnSaveOrUpdate').button('loading').delay(1000).queue(function() {
                var url = vm.smallOrder.id == null ? "small/smallorder/save" : "small/smallorder/update";
                $.ajax({
                    type: "POST",
                    url: baseURL + url,
                    contentType: "application/json",
                    data: JSON.stringify(vm.smallOrder),
                    success: function(r){
                        if(r.code === 0){
                             layer.msg("操作成功", {icon: 1});
                             vm.reload();
                             $('#btnSaveOrUpdate').button('reset');
                             $('#btnSaveOrUpdate').dequeue();
                        }else{
                            layer.alert(r.msg);
                            $('#btnSaveOrUpdate').button('reset');
                            $('#btnSaveOrUpdate').dequeue();
                        }
                    }
                });
			});
		},
		del: function (event) {
			var ids = getSelectedRows();
			if(ids == null){
				return ;
			}
			var lock = false;
            layer.confirm('确定要删除选中的记录？', {
                btn: ['确定','取消'] //按钮
            }, function(){
               if(!lock) {
                    lock = true;
		            $.ajax({
                        type: "POST",
                        url: baseURL + "small/smallorder/delete",
                        contentType: "application/json",
                        data: JSON.stringify(ids),
                        success: function(r){
                            if(r.code == 0){
                                layer.msg("操作成功", {icon: 1});
                                $("#jqGrid").trigger("reloadGrid");
                            }else{
                                layer.alert(r.msg);
                            }
                        }
				    });
			    }
             }, function(){
             });
		},
		getInfo: function(id){
			$.get(baseURL + "small/smallorder/info/"+id, function(r){
                vm.smallOrder = r.smallOrder;
            });
		},
		reload: function (event) {
			vm.showList = true;
			var page = $("#jqGrid").jqGrid('getGridParam','page');
			$("#jqGrid").jqGrid('setGridParam',{ 
                page:page
            }).trigger("reloadGrid");
		}
	}
});

File: src/main/java/com/icloud/modules/small/entity/SmallOrder.java
package com.icloud.modules.small.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import lombok.Data;

import java.io.Serializable;
import java.math.BigDecimal;
import java.util.Date;

/**
 * 订单表
 * 
 * @author zdh
 * @email yyyyyy@cm.com
 * @date 2020-08-13 14:34:02
 */
@Data
@TableName("t_small_order")
public class SmallOrder implements Serializable {
	private static final long serialVersionUID = 1L;

   	   /*  */
       @TableId(value="id", type= IdType.AUTO)
       private Long id;
   	   	   /* 下单渠道 */
       @TableField("channel")
       private String channel;
   	   	   /* 订单号 */
       @TableField("order_no")
       private String orderNo;
        /* 订单类型 订单类型（0团购订单、1普通订单）*/
        @TableField("order_type")
        private String orderType;
   	   	   /* 用户id */
       @TableField("user_id")
       private Long userId;
   	   	   /* 店铺id */
       @TableField("supplier_id")
       private Long supplierId;
   	   	   /* 0生成、处理中、已完成 */
       @TableField("order_status")
       private Integer orderStatus;
   	   	   /* 0未支付、1支付中、2已支付 */
       @TableField("pay_status")
       private Integer payStatus;
   	   	   /* 0生成、1、退款中、2已退款 */
       @TableField("refund_status")
       private Integer refundStatus;
   	   	   /* 0未发货、1发货中、2已发货 */
       @TableField("ship_status")
       private Integer shipStatus;
   	   	   /* 商品(sku)原始价总额 */
       @TableField("sku_original_total_price")
       private BigDecimal skuOriginalTotalPrice;
   	   	   /* 商品(sku)现价总额 */
       @TableField("sku_total_price")
       private BigDecimal skuTotalPrice;
   	   	   /* 运费 */
       @TableField("freight_price")
       private BigDecimal freightPrice;
   	   	   /* 代金券优惠价 */
       @TableField("coupon_price")
       private BigDecimal couponPrice;
   	   	   /* 代金券id */
       @TableField("coupon_id")
       private Long couponId;
   	   	   /* 实付订单金额 */
       @TableField("actual_price")
       private BigDecimal actualPrice;
   	   	   /* 支付金额 */
       @TableField("pay_price")
       private BigDecimal payPrice;
   	   	   /* 支付流水id(本地支付流水) */
       @TableField("pay_id")
       private Long payId;
   	   	   /* 支付渠道名称 */
       @TableField("pay_channel")
       private String payChannel;
   	   	   /* 支付时间(支付成功回调获取的支付时间) */
       @TableField("pay_time")
       private Date payTime;
   	   	   /* 物流方式(物流方式代号) */
       @TableField("ship_code")
       private String shipCode;
   	   	   /* 物流单号 */
       @TableField("ship_no")
       private String shipNo;
   	   	   /* 本地支付单号（用于与第三方支付交互） */
       @TableField("pay_sn")
       private String paySn;
   	   	   /* 发货时间 */
       @TableField("ship_time")
       private Date shipTime;
   	   	   /* 确认收货时间 */
       @TableField("confirm_time")
       private Date confirmTime;
   	   	   /* 省 */
       @TableField("province")
       private String province;
   	   	   /* 市 */
       @TableField("city")
       private String city;
   	   	   /* 县 */
       @TableField("county")
       private String county;
   	   	   /* 详细地址 */
       @TableField("address")
       private String address;
   	   	   /*  */
       @TableField("memo")
       private String memo;
   	   	   /*  */
       @TableField("refund_reason")
       private String refundReason;
   	   	   /*  */
       @TableField("consignee")
       private String consignee;
   	   	   /*  */
       @TableField("phone")
       private String phone;
   	   	   /*  */
       @TableField("create_time")
       private Date createTime;
   	   	   /*  */
       @TableField("modify_time")
       private Date modifyTime;
   	
}
